### ISSUE TITLE
LiberTEM reader?

### ISSUE THREAD

**Issue by ssomnath** [pycroscopy/ScopeReaders#17]
@dxm447 - taking this from workshop chat logs

**Comment by dxm447** [comment]
Yes. [Their supported EM formats](https://libertem.github.io/LiberTEM/formats.html)

**Comment by dxm447** [comment]
Also [NCEMPy has Digital Micrograph import](https://openncem.readthedocs.io/en/latest/ncempy.io.html)
